europeana-i18n.gemspec

```diff
@@ -20,8 +20,9 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.1.0'
 
   s.add_dependency 'rails', '>= 4.0', '< 6.0'
+  s.add_dependency 'i18n-js', '>= 3.0', '< 4.0'
 
-  s.add_development_dependency 'minitest-reporters'
-  s.add_development_dependency 'rake'
+  s.add_development_dependency 'minitest-reporters' 'rake', '~> 0'
```

Comment:
What are the changes on this and the next line for?

Reply:
Without this there is a warning when building the gem.

```
$ gem build europeana-i18n.gemspec 
WARNING:  open-ended dependency on minitest-reportersrake (>= 0, development) is not recommended
  if minitest-reportersrake is semantically versioned, use:
    add_development_dependency 'minitest-reportersrake', '~> 0'
WARNING:  open-ended dependency on rakerake (>= 0, development) is not recommended
  if rakerake is semantically versioned, use:
    add_development_dependency 'rakerake', '~> 0'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
```

Reply:
never mind, that's copy pasted wrong anyway. 
I've updated it with actually usable versions.